### PR TITLE
docs: pull request template

### PR DESCRIPTION
**Pull request by feinoujc**
this will apply to all repos in our org (unless you override it with your own template)

Thoughts welcome

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# [id:XXX](https://ncarb.tpondemand.com/entity/XXX)
```
Comment (feinoujc) [inline .github/PULL_REQUEST_TEMPLATE.md]: id:# is one of the supported formats of the TP/Github integration so I went with that over something like TP-#

**Review by davious — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (davious) [reply .github/PULL_REQUEST_TEMPLATE.md]: just spitballing:
```
description

Target Process Story: [id:XXX](https://ncarb.tpondemand.com/entity/XXX)
```

Comment (feinoujc) [reply .github/PULL_REQUEST_TEMPLATE.md]: sure, that works too

Comment (feinoujc) [reply .github/PULL_REQUEST_TEMPLATE.md]: maybe drop the "Target Process Story:" part? seems redundant to me

Comment (davious) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```
description

[id:XXX](https://ncarb.tpondemand.com/entity/XXX)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# [id:XXX](https://ncarb.tpondemand.com/entity/XXX)
-
 [brief description]
```
Comment (davious) [inline .github/PULL_REQUEST_TEMPLATE.md]: here I paint the shed `description` -- easy to double-click and type over

Comment (feinoujc) [reply .github/PULL_REQUEST_TEMPLATE.md]: sure, or not type in at all. But we can see

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (KOlowoyeye) [reply .github/PULL_REQUEST_TEMPLATE.md]: Works for me, it's similar enough to what we do in the draft repo https://github.com/NCARB/Draft/wiki/Pull-Request-Review-Process#footer

Comment (feinoujc) [reply .github/PULL_REQUEST_TEMPLATE.md]: also FYI on a per repo level you can enable this https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-autolinks-to-reference-external-resources

```
id:<num> -> https://ncarb.tpondemand.com/entity/<num>
```

**Review by davious — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (davious) [reply .github/PULL_REQUEST_TEMPLATE.md]: I was thinking about that.

We could turn that on for all repos and pair down the fluff more, perhaps.

Comment (feinoujc) [reply .github/PULL_REQUEST_TEMPLATE.md]: I wish it was at an org level 😞

**Review by KOlowoyeye — APPROVED** [review]

**Pull request merged**